library/src/main/java/com/novoda/downloadmanager/lib/Query.java

```diff
@@ -31,14 +31,14 @@
      */
     public static final int ORDER_DESCENDING = 2;
 
-    private static final String ORDER_BY_LIVENESS = String.format("CASE batch_status "
+    private static final String ORDER_BY_LIVENESS = String.format("CASE status "
```

Comment:
not entirely sure why I did this change, but given this is querying for downloads rather than batches, it is probably more proper to have it as download status. But yeah, will use the constant :+1: